config/crds/openstackproviderconfig_v1alpha1_openstackclusterproviderstatus.yaml

```diff
@@ -129,6 +129,24 @@ spec:
           description: Network contains all information about the created OpenStack
             Network. It includes Subnets and Router.
           properties:
+            apiServerLoadBalancer:
+              description: Be careful when using APIServerLoadBalancer, because this
+                field is optional and therefore not set in all cases
+              properties:
+                id:
+                  type: string
+                internalIP:
```

Comment:
What's difference between internalIP and IP? could they be the same?

Reply:
internalIP is the VIP of the LoadBalancer, IP is the FloatingIP. So the LB has actually two IPs (to illustrate, an example in our case: IP is 53.58.1.5, internalIP is 10.6.0.20)